orc8r/cloud/docker/build.py

```diff
@@ -99,7 +99,7 @@ def main() -> None:
     elif args.tests:
         _run(['up', '-d', 'postgres_test'])
         _run(['build', 'test'])
-        _run(['run', '--rm', 'test', 'make test'])
+        _run(['run', '--rm'] + _get_mnt_vols(mods) + ['test', 'make test'])
```

Comment:
Okay actually one thing -- we're intentionally *not* mounting any volumes here, because it breaks the "clean room" effect. If you want to just exfiltrate the junit files, mount that directory specifically

Reply:
good point. Will fix :)